### PR TITLE
Test gas schedule change wasm ops

### PR DESCRIPTION
**Pull request by sasurobert**
Test gas schedule change wasm ops

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.0
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
 	github.com/ElrondNetwork/elrond-go-logger v1.0.5
-	github.com/ElrondNetwork/elrond-vm-common v1.2.2
+	github.com/ElrondNetwork/elrond-vm-common v1.2.3-0.20211005101446-652cf8e5d883
```
Comment (iulianpascalau) [inline go.mod:15]: proper release

File: integrationTests/vm/txsFee/scCalls_test.go [hunk 2]
```diff
@@ -229,3 +234,133 @@ func TestScCallOutOfGasShouldConsumeGas(t *testing.T) {
...
+
+		_, err = testContext.Accounts.Commit()
+		require.Nil(t, err)
+
+		intermediateTxs := testContext.GetIntermediateTransactions(t)
+		testIndexer := vm.CreateTestIndexer(t, testContext.ShardCoordinator, testContext.EconomicsData)
+		testIndexer.SaveTransaction(tx, block.TxBlock, intermediateTxs)
+
+		indexerTx := testIndexer.GetIndexerPreparedTransaction(t)
+		// 2 esdt transfers and one api call - + 3
```
Comment (iulianpascalau) [inline integrationTests/vm/txsFee/scCalls_test.go:363]:
```GO
// 2 esdt transfers and one api call - + 3 
```
? Maybe rephrase the comment?

File: factory/processComponents.go [hunk 3]
```diff
@@ -151,6 +149,7 @@ type processComponentsFactory struct {
 	historyRepo            dblookupext.HistoryRepository
 	epochNotifier          process.EpochNotifier
 	importHandler          update.ImportHandler
+	arwenChangeLocker      common.Locker
```
[the remark this answers is not in the document]
Comment (sasurobert) [reply factory/processComponents.go:152]: done
